docs/core/docker/building-net-docker-images.md

```diff
@@ -2,224 +2,251 @@
+docker run -it --rm --name aspnetcore_sample aspnetapp
 ```
 
-Another way to generate the image and run the application within the Docker container is to open the application in Visual Studio Code and use the debugging tools. 
+> [!IMPORTANT]
+> You must navigate to the **container IP address** (as opposed to http://localhost) in your browser directly when using Windows containers. You can get the IP address of your container with the following steps:
 
-Select the debugging icon in the View Bar on the left side of Visual Studio Code.
+* Open up another command prompt.
+* Run `docker ps` to see your running containers. The "aspnetcore_sample" container should be there.
+* Run `docker exec` aspnetcore_sample ipconfig.
+* Copy the container IP address and paste into your browser (for example, 172.29.245.43).
 
-![vscode debugging icon](./media/building-net-docker-images/debugging_debugicon.png)
+> [!Note]
+> Docker exec supports identifying containers with name or hash. The name (aspnetcore_sample) is used in our example.
 
-Then tap the play icon or F5 to generate the image and start the application within the container. The Web API will be launched using your default web browser at http://localhost:5000.
+See the following example of how to get the IP address of a running Windows container.
 
-![VSCode Docker Tools Debug](./media/building-net-docker-images/docker-tools-vscode-f5.png)
+```console
+docker exec aspnetcore_sample ipconfig
 
-You may set break points in your application, step through, etc. just as if the application was running locally on your development machine as opposed to inside the container. The benefit to debugging within the container is this is the same image that would be deployed to a production environment.
+Windows IP Configuration
 
-Creating the release or production image requires simply running the command from the terminal passing the `release` environment name.
+Ethernet adapter Ethernet:
 
-```bash
-./dockerTask build release
+   Connection-specific DNS Suffix  . : contoso.com
+   Link-local IPv6 Address . . . . . : fe80::1967:6598:124:cfa3%4
+   IPv4 Address. . . . . . . . . . . : 172.29.245.43
+   Subnet Mask . . . . . . . . . . . : 255.255.240.0
+   Default Gateway . . . . . . . . . : 172.29.240.1
 ```
 
-The command creates the image based on the smaller **microsoft/dotnet:core** base image, [EXPOSE](https://docs.docker.com/engine/reference/builder/#/expose) port 5000, sets the [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#/entrypoint) for `dotnet api.dll` and copies it to the `/app` directory. There is no debugger, SDK or `dotnet restore` ([see note](#dotnet-restore-note)) resulting in a much smaller image. The image is named **api** with a `TAG` of **latest**.
+> [!Note]
+> Docker exec runs a new command in a running command. For more information, see the [docker exec reference](https://docs.docker.com/engine/reference/commandline/exec/) on command-line parameters.
```

Comment:
typo: "in a running command" in the first sentence should be "in a running container."